src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -88,19 +92,21 @@ private Manifest BuildCore(DocumentBuildParameters parameters)
                         ManifestFile = "manifest",
                         XRefSpecMapFile = "xrefspecmap",
                         BuildModelManifestFile = "buildmodelmanifest",
-                        PostBuildModelManifestFile = "postbuildmodelmanifest",
+                        BuildMessageFile = "buildmessage",
```

Comment:
hard code message file name, will be remove.

---
In reply to: [79969979](https://github.com/dotnet/docfx/pull/724#discussion_r79969979) [](ancestors = 79969979)